internal/rbd/encryption.go

```diff
@@ -184,7 +184,7 @@ func (ri *rbdImage) repairEncryptionConfig(dest *rbdImage) error {
 			dest.conn = ri.conn.Copy()
 		}
 
-		return ri.copyEncryptionConfig(dest, false)
+		return ri.copyEncryptionConfig(dest, true)
```

Comment:
Destination pvc in case of restore/clone will have possibly different encryption config already initialized.